### PR TITLE
install internal gems; update repository URL

### PR DESCRIPTION
**Pull request by ericcf**
* bit_player and git_tagger gems
* repository moved from 'cbitstech' to 'NU-CBITS'

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,13 +29,9 @@ gem "event_capture", "~> 0.1.2"
 gem "think_feel_do_engine", "~> 3.15"
 gem "think_feel_do_dashboard", "~> 1.1"
 gem "bit_core", "~> 1.4"
-gem "bit_player",
-    tag: "0.4.10",
-    git: "https://github.com/cbitstech/bit_player.git"
+gem "bit_player", "~> 0.4.10"
```
Comment (michaelwehrley) [inline Gemfile]: I would vote for "~> 0.4"

**Comment by michaelwehrley** [comment]
👍

File: Gemfile (hunk 1 above)
Comment (ericcf) [inline Gemfile]: bit_player 0.5 is tied to Rails 5, so not yet...

Comment (michaelwehrley) [inline Gemfile]: 👀

**Pull request merged**